storage/innobase/handler/ha_innopart.cc

```diff
@@ -1111,13 +1106,7 @@ ha_innopart::open(
 
 			no_tablespace = false;
 
-<<<<<<< HEAD
-	} else if (ib_table->file_unreadable) {
-||||||| 0cd98bdf981
-	} else if (ib_table->ibd_file_missing) {
-=======
-		} else if (ib_table->ibd_file_missing) {
->>>>>>> c94ce787737f9a6c9a31abe22bee02866698885b^
+		} else if (ib_table->file_unreadable || ib_table->corrupted) {
```

Comment:
Shouldn't it be `ibd_file_missing || corrupted`?

Reply:
Here ibd_file_missing was changed to file_unreadable some time ago in https://github.com/percona/percona-server/commit/c7f44ee3974df2ffe9938f673d25cd5e47beeae1. Last upstream update didn't change logic here. I've moved checkup for ib_table->corrupted here as it didn't work any more after changes in https://github.com/mysql/mysql-server/commit/4e3e5030a27f9c75a52c546a961aafddb11571c6.